compiler/src/dotty/tools/dotc/printing/SyntaxHighlighting.scala

```diff
@@ -81,6 +82,10 @@ object SyntaxHighlighting {
         }
       }
 
+      scanner.commentPositions.foreach {
+        highlightPosition(_,CommentColor)
+      }
```

Comment:
nitpick: we usually don't use the underscore syntax with curly braces. I suggest writing
```scala
for (pos <- scanner.commentPositions)
  highlightPosition(pos, CommentColor)
```
or
```scala
scanner.commentPositions.foreach { pos =>
  highlightPosition(pos, CommentColor)
}
```
or
```scala
scanner.commentPositions.foreach(highlightPosition(_, CommentColor))
```

I personally prefer the first one

Reply:
I have selected the first approach only